rotkehlchen/db/history_events.py

```diff
@@ -628,12 +628,15 @@ def get_history_events_count(
             )
 
             free_query_group_by = maybe_filter_ignore_asset(query_filter)
+            group_by_query = ''
+            if group_by_event_ids:
+                group_by_query = 'GROUP BY event_identifier ORDER BY timestamp DESC, sequence_index ASC LIMIT ?'  # noqa: E501
```

Comment:
how does this change affect the frontend? Does it break something in their assumptions? It should be documented in the PR description if it is changing the default behaviour

Reply:
I tested it by running the app, and all the paginated events were showing as expected. Also, I've asked the frontend devs, if they ever use `group_by_event_ids: false` on this API.

Reply:
@tewshi confirmed that it's not used.